src/color.rs

```diff
@@ -612,3 +957,134 @@ impl ShadingType {
         }
     }
 }
+
+/// Writer for the _separation information dictionary_. PDF 1.3+.
+///
+/// This struct is created by [`Catalog::separation_info`].
+pub struct SeparationInfo<'a> {
+    dict: Dict<'a>,
+}
+
+writer!(SeparationInfo: |obj| Self { dict: obj.dict() });
+
+impl SeparationInfo<'_> {
+    /// Write the `/Pages` attribute. Required.
+    ///
+    /// This indicates all page dictionaries in the document that represent
+    /// separations of the same page and shall be rendered together.
+    pub fn pages(&mut self, pages: impl IntoIterator<Item = Ref>) -> &mut Self {
+        self.dict.insert(Name(b"Pages")).array().typed().items(pages);
+        self
+    }
+
+    /// Write the `/DeviceColorant` attribute as a name. Required.
+    ///
+    /// The name of the device colorant that corresponds to the separation.
+    pub fn device_colorant(&mut self, colorant: Name) -> &mut Self {
+        self.dict.pair(Name(b"DeviceColorant"), colorant);
+        self
+    }
+
+    /// Write the `/DeviceColorant` attribute as a string. Required.
+    ///
+    /// The name of the device colorant that corresponds to the separation.
+    pub fn device_colorant_str(&mut self, colorant: &str) -> &mut Self {
```

Comment:
what does this do compared to specifying a `name`?

Reply:
Don't know, allows Unicode characters? PDF allows both